static/bidder-info/adinify.yaml

```diff
@@ -0,0 +1,5 @@
+endpoint: "https://adinify-prebid.attekmi.com/pbserver/?seat={{.AccountID}}&token={{.SourceId}}"
+aliasOf: "smarthub"
+userSync:
+  supports:
+    - redirect
```

Comment:
You can omit this since this is exactly what you have for smarthub; it will be inherited.

Reply:
Ok I removed it